agent/util-scripts/pbench-tool-meister-stop

```diff
@@ -18,10 +18,15 @@ import time
 from argparse import ArgumentParser, Namespace
 from pathlib import Path
 
-from pbench.agent.constants import def_redis_port, cli_tm_channel_prefix
+from pbench.agent.constants import cli_tm_channel_prefix
```

Comment:
`pbench-tool-meister-start` references `def_redis_port` in its help text.  Should `pbench-tool-meister-stop` be doing the same?  Or, better yet, should the two tools be sharing their argument definitions for the pieces which should be uniform between them?